### DISCUSSION TITLE
Headset purchased or fabricated has incorrect TeamID

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#14998] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

headset has incorrect TeamID and can not be used to communicate with other teammate

### Reproduction steps

1. purchase a headset and try to use to chat with others
2. start fabricating a headset and undock
3. try to use fabricated headset to chat with others

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.6.18.1 (Unto the Breach Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! I'm afraid I could not find a way to fully reproduce the issue. To double-check, you're not using any mods?

https://github.com/user-attachments/assets/c749cf28-4928-488a-9d25-1062620e9cb8

I got it to happen when purchasing the headset with immediate delivery though, that seems to be clearly bugged.

**Reply by A2ur3** [reply]
start fabrication then undock. when fabrication completed you will get bugged headset
fabricator is unable to remember who fabricated the item across rounds
since the fabricator is in player submarine the headset should set default TeamID as Team1

**Comment by Regalis11** [comment]
This has now been fixed in our private repo, fix coming up in the next unstable build and the upcoming release.
